### PR TITLE
Update Netherlands.csv

### PR DESCRIPTION
**Pull request by loekvangool**

### CODE REVIEW

**Comment by lucasrodes** [comment]
Hi @loekvangool,  thanks for sharing your findings. 

Note that the way this project is built PRs for data updates is not supported. However, reporting new data available is pretty much appreciated 😄 . More info in #230 and #250.

Two brief comments on your modifications:

- Could you provide a reference for your vaccination data modifications?
- I assume 163.931 means 163931. Keeping a point makes it be interpreted as a string instead of numeric.

**Comment by loekvangool** [comment]
>I assume 163.931 means 163931. Keeping a point makes it be interpreted as a string instead of numeric.@lucasrodes d'oh that's embarrassing

**Comment by lucasrodes** [comment]
> There is a URL present in the line, is that what you mean? Pasting it here again for clarity: https://coronadashboard.rijksoverheid.nl/landelijk/vaccinaties

@edomt We might consider adding an automation script for this URL. I can work on this later if not done before 😄

**Comment by EwoutH** [comment]
I added an overview of our latest data-sources and when they are updated here: https://github.com/owid/covid-19-data/issues/374

File: public/data/vaccinations/country_data/Netherlands.csv [hunk 1]
```diff
@@ -11,3 +11,4 @@ Netherlands,2021-01-15,Pfizer/BioNTech,https://twitter.com/hugodejonge/status/13
 Netherlands,2021-01-17,Pfizer/BioNTech,https://www.rivm.nl/covid-19-vaccinatie/cijfers-vaccinatieprogramma,77000,77000,0
 Netherlands,2021-01-20,Pfizer/BioNTech,https://twitter.com/hugodejonge/status/1351945435452219394,100000,100000,0
 Netherlands,2021-01-22,"Moderna, Pfizer/BioNTech",https://twitter.com/hugodejonge/status/1352641502892204040,135000,135000,0
+Netherlands,2021-01-25,"Moderna, Pfizer/BioNTech",https://coronadashboard.rijksoverheid.nl/landelijk/vaccinaties,163931,163931,0
```
Comment (EwoutH) [inline public/data/vaccinations/country_data/Netherlands.csv:14]: Use English page of our Coronavirus Dashboard.
```suggestion
Netherlands,2021-01-25,"Moderna, Pfizer/BioNTech",https://coronadashboard.government.nl/landelijk/vaccinaties,163931,163931,0
```

**Review by EwoutH — APPROVED** [review]

**Comment by EwoutH** [comment]
You could also add today's data (173638).

**Comment by EwoutH** [comment]
It's now incorrect, 2021-01-25 should be 163931 and 2021-01-26 173638

**Comment by loekvangool** [comment]
You know I'm gonna stick to PHP 👶

**Pull request closed without merging**